### PR TITLE
add link to release notes on first page of manual

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

File: doc/make.jl [hunk 1]
```diff
@@ -38,8 +38,16 @@ for dir in readdir(STDLIB_DIR)
     end
 end
 
+# Generate a suitable markdown file from NEWS.md and put it in src
+str = read(joinpath(@__DIR__, "..", "NEWS.md"), String)
+splitted = split(str, "<!--- generated by NEWS-update.jl: -->")
```
Comment (fredrikekre) [inline doc/make.jl]: Should we also split out the previous versions release notes that might not have been removed from NEWS.md yet?

Comment (KristofferC) [reply doc/make.jl]: I think it is fine to have them there for "latest" versions of the manual. For stable the old stuff will have been removed.

File: doc/src/index.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-# Julia Documentation
+# Julia 0.7 Documentation
```
Comment (fredrikekre) [inline doc/src/index.md]: Worth generating this from `VERSION`?

File: doc/src/index.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
...
+
+Welcome to the documentation for Julia 0.7.
+
+Please read the [release notes](NEWS.md) to see what has changed since Julia 0.6.
```
Comment (fredrikekre) [inline doc/src/index.md]: `... what has changed since Julia 0.6.` -> `... what has changed since the last release.` ?

**Review by fredrikekre — APPROVED** [review]
This is a much nicer way to read NEWS.md :tada:

File: doc/src/index.md (hunk 2 above)
Comment (KristofferC) [reply doc/src/index.md]: I thought about it but felt it was an unnecessary complication. We could do it later if someone feels like it.

**Pull request merged**